### PR TITLE
Addresses #3216, wrong OS:Daylighting:Control rotation angle translated

### LINKED ISSUE #3216: Wrong OS:Daylighting:Control rotation angle translated
OpenStudio only translates the OS:Daylighting:Control input 'theta rotation around the Y-Axis' (the other rotations are ignored), and sets that value to the Daylighting:ReferencePoint input "Glare Calculation Azimuth Angle of View Direction Clockwise from Zone y-Axis". This happens here:
https://github.com/NREL/OpenStudio/blob/develop/openstudiocore/src/energyplus/ForwardTranslator/ForwardTranslateThermalZone.cpp#L457-L469

In OpenStudio, 'rotation around the Y-axis' (the zone north axis) is rotation in the vertical plane. This can be seen in Sketchup if you place a daylighting control object and change the theta value. The expected Daylighting:ReferencePoint EnergyPlus input, however, is "the angle, measured clockwise in **the horizontal plane**, between the zone y-axis and the occupant view direction." (ref: https://bigladdersoftware.com/epx/docs/8-9/input-output-reference/group-daylighting.html#field-glare-calculation-azimuth-angle-of-view-direction-clockwise-from-zone-y-axis).

In aircraft terms, for a body facing north, EnergyPlus is expecting the yaw angle, but OpenStudio is translating the roll angle. 

To reflect the expected input, OpenStudio should be translating the phiRotationAroundZAxis value.

### PR DESCRIPTION
**Pull request by joseph-robertson**
Pull request overview
---------------------

 - Fixes #3216
 - Removes conversion of minus rad to deg in ft (and minus deg to rad in rt)
 - Add ft and rt tests that address this issue

Please read [OpenStudio Pull Requests](https://github.com/NREL/OpenStudio/wiki/OpenStudio-Pull-Requests) to better understand the OpenStudio Pull Request protocol.

### Pull Request Author

Add to this list or remove from it as applicable.  This is a simple templated set of guidelines.

 - [ ] Model API Changes / Additions
 - [ ] Any new or modified fields have been implemented in the EnergyPlus ForwardTranslator (and ReverseTranslator as appropriate)
 - [ ] Model API methods are tested (in `src/model/test`)
 - [ ] EnergyPlus ForwardTranslator Tests (in `src/energyplus/Test`)
 - [ ] If a new object or method, added a test in NREL/OpenStudio-resources: Add Link
 - [ ] If needed, added VersionTranslation rules for the objects (`src/osversion/VersionTranslator.cpp`)
 - [ ] Checked behavior in OpenStudioApplication, adjusted policies as needed (`src/openstudio_lib/library/OpenStudioPolicy.xml`)
 - [ ] Verified that C# bindings built fine on Windows, partial classes used as needed, etc.
 - [ ] All new and existing tests passes
 - [ ] If methods have been deprecated, update rest of code to use the new methods

**Labels:**

 - [ ] If change to an IDD file, add the label `IDDChange`
 - [ ] If breaking existing API, add the label `APIChange`
 - [ ] If deemed ready, add label `Pull Request - Ready for CI` so that CI builds your PR

### Review Checklist

This will not be exhaustively relevant to every PR.
 - [ ] Perform a Code Review on GitHub
 - [ ] Code Style, strip trailing whitespace, etc.
 - [ ] All related changes have been implemented: model changes, model tests, FT changes, FT tests, VersionTranslation, OS App
 - [ ] Labeling is ok
 - [ ] If defect, verify by running develop branch and reproducing defect, then running PR and reproducing fix
 - [ ] If feature, test running new feature, try creative ways to break it
 - [ ] CI status: all green or justified

### CODE REVIEW

**Comment by jmarrec** [comment]
@joseph-robertson There's a minor merge conflict that I'll resolve.

File: src/energyplus/ForwardTranslator/ForwardTranslateThermalZone.cpp [hunk 1]
```diff
@@ -467,12 +467,12 @@ boost::optional<IdfObject> ForwardTranslator::translateThermalZone( ThermalZone
...
 
-      if (primaryDaylightingControl->phiRotationAroundZAxis() != 0.0){
-        LOG(Warn, "Rotation of " << primaryDaylightingControl->phiRotationAroundZAxis() << " degrees about Z axis not mapped for OS:Daylighting:Control " << primaryDaylightingControl->name().get());
+      if (primaryDaylightingControl->thetaRotationAroundYAxis() != 0.0){
+        LOG(Warn, "Rotation of " << primaryDaylightingControl->thetaRotationAroundYAxis() << " degrees about Y axis not mapped for OS:Daylighting:Control " << primaryDaylightingControl->name().get());
       }
 
       // glare
-      double glareAngle = -openstudio::radToDeg(primaryDaylightingControl->thetaRotationAroundYAxis());
+      double glareAngle = primaryDaylightingControl->phiRotationAroundZAxis();
```
[the remark this answers is not in the document]
Comment (jmarrec) [reply src/energyplus/ForwardTranslator/ForwardTranslateThermalZone.cpp:475]: I think the "-" was needed actually. @eringold has a file that is throwing in 3.2.1 but worked as 2.9.0, and I tracked it down to here.

```
   ** Severe  ** <root>[Daylighting:Controls][TZ_L6_BKS_Corridor/Toilets DaylightingControls][glare_calculation_azimuth_angle_of_view_direction_clockwise_from_zone_y_axis] - "-0.000000" - Expected number greater than or equal to 0.000000
```

[the remark this answers is not in the document]
Comment (macumber) [reply src/energyplus/ForwardTranslator/ForwardTranslateThermalZone.cpp:475]: Agree that the negative sign should stay.  If E+ requires rotation angle to be positive you could also do a:

```
if (angle < 0){
  angle += 360;
}
```

[the remark this answers is not in the document]
Comment (jmarrec) [reply src/energyplus/ForwardTranslator/ForwardTranslateThermalZone.cpp:475]: By "also" you mean "in addition" right?  

If I follow correctly, openstudio's convention for phi is counterclockwise, which is the opposite of the convention for phi in E+ (clockwise)?

https://bigladdersoftware.com/epx/docs/9-5/input-output-reference/group-daylighting.html#field-glare-calculation-azimuth-angle-of-view-direction-clockwise-from-zone-y-axis

do you recall if it's documented somewhere (at least in code comments somewhere)?

Only thing I could find is this: https://github.com/NREL/OpenStudio/blob/057a86b6f80ad651ed5537b65c3af23a0308aee8/src/energyplus/GeometryTranslator.cpp#L350

[the remark this answers is not in the document]
Comment (macumber) [reply src/energyplus/ForwardTranslator/ForwardTranslateThermalZone.cpp:475]: @jmarrec yes I meant in addition, if after applying the negative sign the result for E+ is still negative you can add 360.  The angle mod 360 might also work, you could try it with a unit test.  If you use the right hand rule, https://en.wikipedia.org/wiki/Right-hand_rule, with y pointing North and x pointing East, then a positive rotation around the z-axis (pointing up) is counter-clockwise when viewed from above.  The E+ convention of positive rotation in the clockwise direction is backwards to most conventions.

**Pull request merged**